salt/modules/solr.py

```diff
@@ -1026,12 +1007,10 @@ def signal(signal=None):
     # Give a friendly error message for invalid signals
     # TODO: Fix this logic to be reusable and used by apache.signal
     if signal not in valid_signals:
-        msg = valid_signals[:-1] + ("or {0}".format(valid_signals[-1]),)
-        return "{0} is an invalid signal. Try: one of: {1}".format(
-            signal, ", ".join(msg)
-        )
+        msg = valid_signals[:-1] + ("or {}".format(valid_signals[-1]),)
+        return "{} is an invalid signal. Try: one of: {}".format(signal, ", ".join(msg))
```

Comment:
```suggestion
        return "{} is an invalid signal. Try: one of: {} or {}".format(
            signal,
            ", ".join(valid_signals[:-1]),
            valid_signals[-1]
        )
```
Since we're changing the file